AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/TestSupport/SyncEngineTestBase.swift

```diff
@@ -41,11 +41,14 @@ class SyncEngineTestBase: XCTestCase {
 
     // MARK: - Setup
 
+    override func setUp() {
+        Amplify.Logging.logLevel = .verbose
+    }
+
```

Comment:
according to https://developer.apple.com/documentation/xctest/xctest/1500341-setup XCTest calls setUpWithError(), followed by setUp().. this seems like hidden information, unless we have a reason to have two set up methods, would be clearer to move the logLevel set to the last line of `setUpWithError`